src/main/java/ua/softserveinc/tc/validator/ChildValidator.java

```diff
@@ -46,14 +49,12 @@ public void validate(Object o, Errors errors) {
         String fName = kidToValidate.getFirstName();
         String lName = kidToValidate.getLastName();
 
-        if ((fName.length() < 2) || (lName.length() < 2)) {
+        if(fName.length() < MIN_NAME_CHARACTER) {
+            errors.rejectValue(ValidationConstants.FIRST_NAME, ValidationConstants.NAME_NOT_EDITTED);
+        } else if(lName.length() < MIN_NAME_CHARACTER) {
+            errors.rejectValue(ValidationConstants.LAST_NAME, ValidationConstants.NAME_NOT_EDITTED);
+        }
```

Comment:
Validation will happen only in one field. When  (fName.length < MIN_NAME_CHARACTER) gives true, else statement never happened even if lName.length < MIN_NAME_CHARACTER)
 I think that  you should write two independent if without else.
Example:
 if(fName.length() < MIN_NAME_CHARACTER){
// code
}
 if(lName.length() < MIN_NAME_CHARACTER){
// code
}